lib/get-pages-data.js

```diff
@@ -8,7 +8,11 @@ module.exports = (stories) => {
     .map(story => ({
       content: addComponentSlug(story.content.content),
       title: story.name,
-      slug: story.slug,
+      slug: correctHomepageSlug(story.slug),
       full_slug: story.full_slug
     }))
 }
+
+function correctHomepageSlug(slug) {
+  return slug.includes('home') ? '' : slug
```

Comment:
"If the slug includes 'home', return nothing after the "/".
Else, return the slug that comes after the "/".
This way the homepage won't have to be called cmd-digitalplayground.nl/home, but just cmd-digitalplayground.nl/."